src/local/butler/reproduce.py

```diff
@@ -272,7 +279,15 @@ def _reproduce_crash(testcase_id, build_directory):
 
 def execute(args):
   """Attempt to reproduce a crash then report on the result."""
-  result = _reproduce_crash(args.testcase, args.build_dir)
+  try:
+    result = _reproduce_crash(args.testcase, args.build_dir)
+  except ReproduceToolException as exception:
+    print(exception)
```

Comment:
Why not just raise ?

Reply:
It just seems like cleaner output. The cases this catches are supposed to be unrecoverable errors that we've handled explicitly.